### ISSUE TITLE
Import error: `symbol not found in flat namespace '_CFRelease'`

### ISSUE THREAD

**Issue by elliottower** [deepmind/lab2d#24]
Hi, I've been getting this error when trying to install on an intel mac. Previously I was able to install but I deleted the virtual environment and need to install it to a new one.
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/dmlab2d/__init__.py", line 21, in <module>
    import dmlab2d.dmlab2d_pybind as dmlab2d_pybind
ImportError: dlopen(/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/dmlab2d/dmlab2d_pybind.so, 0x0002): symbol not found in flat namespace '_CFRelease'
```

**Comment by tkoeppe** [comment]
I think this means you're missing the "Core Foundation" framework. You might need to add a linkopt like `-framework CoreFoundation` somewhere.

See also https://github.com/deepmind/lab2d/issues/23#issuecomment-1497779383.

**Comment by elliottower** [comment]
I'm not on an M1/M2 mac so not sure if that issue is relevant to me. Tried what you suggested and it didn't change anything.

**Comment by tkoeppe** [comment]
I think the frameworks are about being on MacOS, not specifically on which hardware. I think `_CFRelease` comes from the Core Foundation framework? But I'm not exactly sure at which point that needs to be linked I'm afraid.

**Comment by tkoeppe** [comment]
Coming to think of it, could this be the "chained fixups" bug, https://github.com/bazelbuild/bazel/issues/16413? It's present in XCode 14.1 and 14.2, but fixed in 14.3 I believe.

**Comment by tkoeppe** [comment]
I think we can call this solved:

* Either upgrade to XCode 14.3 or newer, or
* Use the workaround `--linkopt=-Wl,-no_fixup_chains`.
